### ISSUE TITLE
[Bug] Harvester banned on local network

### ISSUE THREAD

**Issue by softwareuser99** [Chia-Network/chia-blockchain#15788]
### What happened?

**Encountered behaviour:**

As of the installation of below version, a harvester on my farm constantly encounters being banned.

```
2023-07-16T... harvester harvester               : INFO     Failed to connect to PeerInfo(_ip=IPv4Address('10.10.10.85'), _port=8447)

2023-07-16T...  harvester harvester_server        : WARNING  Peer 10.10.10.85 is still banned, not connecting to it

```

**Expected behaviour:**

Harvester would connect to main farmer without issues.

**Environment:**

- 5 harvesters, 1 main farm
- Ubuntu 22.04 on all machines.
- 100% C7 plots, no OG plots
- All machines running the version in question
- All machines using GPU harvesting (also reocgnized in the GUI of the main farmer)
- During upgrade, all .chia folders were deleted and recreated with chia init
- SSL certificates were re-propagated in the LAN and installed with chia init -c ca

Thanks for having a look into this and happy to provide more detail.

### Version

2.0.0b5-dev76

### What platform are you using?

Linux

### What ui mode are you using?

CLI

### Relevant log output

```shell
There is no relevant log output beyond what is cited above.
```

**Comment by urpayn** [comment]
I am encountering this on my harvesters with 2.0.0rc4 on them. the farmer/node is also 2.0.0rc4
I recreated the config file on each manually as recommended during the update.
Some harvesters I seem to have been able to get them going after repeating the process a few times.
I still have some that wont stay connected.
`2023-08-12T20:45:01.483 harvester chia.harvester.harvester: INFO     0 plots were eligible for farming 4b0da3a1f1... Found 0 proofs. Time: 0.01909 s. Total 99 plots
2023-08-12T20:45:10.837 harvester chia.harvester.harvester: INFO     0 plots were eligible for farming 4b0da3a1f1... Found 0 proofs. Time: 0.01815 s. Total 99 plots
2023-08-12T20:45:19.640 harvester chia.harvester.harvester: INFO     0 plots were eligible for farming 4b0da3a1f1... Found 0 proofs. Time: 0.02233 s. Total 99 plots
2023-08-12T20:45:28.327 harvester chia.harvester.harvester: INFO     0 plots were eligible for farming 4b0da3a1f1... Found 0 proofs. Time: 0.02270 s. Total 99 plots
2023-08-12T20:45:37.140 harvester chia.harvester.harvester: INFO     0 plots were eligible for farming 4b0da3a1f1... Found 0 proofs. Time: 0.02219 s. Total 99 plots
2023-08-12T20:45:46.932 harvester chia.harvester.harvester: INFO     0 plots were eligible for farming 4b0da3a1f1... Found 0 proofs. Time: 0.01930 s. Total 99 plots
2023-08-12T20:45:55.742 harvester chia.harvester.harvester: INFO     0 plots were eligible for farming 4b0da3a1f1... Found 0 proofs. Time: 0.01880 s. Total 99 plots
2023-08-12T20:46:04.429 harvester chia.harvester.harvester: INFO     0 plots were eligible for farming 4b0da3a1f1... Found 0 proofs. Time: 0.02498 s. Total 99 plots
2023-08-12T20:46:20.506 harvester harvester_server        : ERROR    Exception:  <class 'asyncio.exceptions.TimeoutError'>, closing connection PeerInfo(_ip=IPv4Address('192.168.1.126'), _port=8447). Traceback (most recent call last):
  File "chia/server/ws_connection.py", line 420, in _api_call
  File "asyncio/tasks.py", line 501, in wait_for
asyncio.exceptions.TimeoutError

2023-08-12T20:46:20.512 harvester harvester_server        : WARNING  Banning 192.168.1.126 for 10 seconds
2023-08-12T20:46:20.512 harvester harvester_server        : INFO     Connection closed: 192.168.1.126, node id: 45a728385007c49abd50e448d890f9f93aaa0e9a9b951ca30c9ac068254ff023
2023-08-12T20:46:20.513 harvester chia.harvester.harvester: INFO     peer disconnected PeerInfo(_ip=IPv4Address('192.168.1.126'), _port=8447)
2023-08-12T20:46:22.170 harvester harvester_server        : WARNING  Peer 192.168.1.126 is still banned, not connecting to it
2023-08-12T20:46:22.173 harvester harvester               : INFO     Failed to connect to PeerInfo(_ip=IPv4Address('192.168.1.126'), _port=8447)
2023-08-12T20:46:25.176 harvester harvester_server        : WARNING  Peer 192.168.1.126 is still banned, not connecting to it
2023-08-12T20:46:25.177 harvester harvester               : INFO     Failed to connect to PeerInfo(_ip=IPv4Address('192.168.1.126'), _port=8447)
2023-08-12T20:46:28.180 harvester harvester_server        : WARNING  Peer 192.168.1.126 is still banned, not connecting to it
2023-08-12T20:46:28.182 harvester harvester               : INFO     Failed to connect to PeerInfo(_ip=IPv4Address('192.168.1.126'), _port=8447)
2023-08-12T20:46:31.208 harvester harvester_server        : INFO     Connected with farmer PeerInfo(_ip=IPv4Address('192.168.1.126'), _port=8447)
2023-08-12T20:46:31.226 harvester chia.plotting.cache     : INFO     Loaded 97360 bytes of cached data
2023-08-12T20:46:31.262 harvester chia.plotting.cache     : INFO     Parsed 99 cache entries in 0.04s
2023-08-12T20:46:34.378 harvester chia.harvester.harvester: INFO     0 plots were eligible for farming 4b0da3a1f1... Found 0 proofs. Time: 0.01990 s. Total 99 plots
2023-08-12T20:46:44.215 harvester chia.harvester.harvester: INFO     0 plots were eligible for farming 4b0da3a1f1... Found 0 proofs. Time: 0.01930 s. Total 99 plots
2023-08-12T20:46:50.536 harvester chia.plotting.util      : WARNING  Directory: /srv/plots does not exist.
2023-08-12T20:46:50.538 harvester chia.plotting.util      : WARNING  Directory: /opt/plots2 does not exist.
2023-08-12T20:46:50.539 harvester chia.plotting.util      : WARNING  Directory: /srv/plots2 does not exist.
2023-08-12T20:46:50.541 harvester chia.harvester.harvester: INFO     _plot_refresh_callback: event started, loaded 0, removed 0, processed 0, remaining 99, duration: 0.00 seconds, total plots: 99
2023-08-12T20:46:50.613 harvester chia.harvester.harvester: INFO     _plot_refresh_callback: event done, loaded 0, removed 0, processed 99, remaining 0, duration: 0.06 seconds, total plots: 99
2023-08-12T20:46:52.664 harvester chia.harvester.harvester: INFO     0 plots were eligible for farming 4b0da3a1f1... Found 0 proofs. Time: 0.02501 s. Total 99 plots
2023-08-12T20:47:02.515 harvester chia.harvester.harvester: INFO     0 plots were eligible for farming 4b0da3a1f1... Found 0 proofs. Time: 0.02233 s. Total 99 plots
`
